queries/injections.scm

```diff
@@ -0,0 +1,4 @@
+[
+  (line_comment)
+  (block_comment)
+] @comment
```

Comment:
pony docstrings usually contain markdown. Should we write some rules to detect pony markdown code-listings and detect and mark them here?

Reply:
This is tough, we could potentially do that by just injecting markdown when triple backticks are detected